teps/0141-platform-context-variables.md

```diff
@@ -224,190 +232,319 @@ The downside of this approach is that Tasks can't easily reference the values of
 
 ## Proposal
 
-<!--
-This is where we get down to the specifics of what the proposal actually is.
-This should have enough detail that reviewers can understand exactly what
-you're proposing, but should not include things like API designs or
-implementation. The "Design Details" section below is for the real
-nitty-gritty.
--->
+There are several components to this proposal:
 
-### Notes and Caveats
+- New TaskRun and PipelineRun syntax allowing TaskRun and PipelineRun authors
+to reference platform-provided context variables
+- The mechanism platform builders use to specify the values of each context variable per Run
+- The way these variables are surfaced in the TaskRun and PipelineRun status and used in provenance generation
 
-<!--
-(optional)
+## Design Details
 
-Go in to as much detail as necessary here.
-- What are the caveats to the proposal?
-- What are some important details that didn't come across above?
-- What are the core concepts and how do they relate?
--->
+### Stability level
 
+This feature will start as an alpha feature and will be enabled by a new feature flag "enable-platform-context-variables".
 
-## Design Details
+### TaskRun and PipelineRun syntax
```

Comment:
Hi @afrittoli @pritidesai, it would be great if you could take a look at this section first? I want to get some early feedback on whether this proposed solution makes sense, or we need to make some changes, or seek alternatives. 
Thanks!